### PR TITLE
Fix health_check bug

### PR DESCRIPTION
**Pull request by cshcomcom**

### CODE REVIEW

File: worker_node/job/send.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ class Send extends Base {
         url: '/health_check',
       });
 
-      if (res.data === 'true') {
+      if (res.data === 'true' || res.data === true) {
```
Comment (platfowner) [inline worker_node/job/send.js:73]: minor: what about borrowing a util function?
https://github.com/ainblockchain/ain-blockchain/blob/master/db/rule-util.js#L109

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments
Thanks

File: worker_node/job/send.js (hunk 1 above)
Comment (cshcomcom) [reply worker_node/job/send.js]: Looks good! Done

**Comment by cshcomcom** [comment]
Thanks for the quick review! Now merging

**Pull request merged**